src/main/java/org/opentripplanner/graph_builder/module/GtfsModule.java

```diff
@@ -125,6 +127,8 @@ public void buildGraph(
 
         OtpTransitService transitModel = builder.build();
 
+        hasTransit = transitModel.getAllServiceIds().size() > 0;
```

Comment:
This code is repeated. Can you move it inside `OtpTransitService`?

Reply:
Done